app/views/user_mailer/_issue_details.text.erb

```diff
@@ -37,6 +37,13 @@ See doc/COPYRIGHT.rdoc for more details.
 <%= WorkPackage.human_attribute_name(:responsible)   %>: <%= issue.responsible   %>
 <%= WorkPackage.human_attribute_name(:category)      %>: <%= issue.category      %>
 <%= WorkPackage.human_attribute_name(:fixed_version) %>: <%= issue.fixed_version %>
+<%= WorkPackage.human_attribute_name(:start_date)    %>: <%= issue.start_date    %>
+<%= WorkPackage.human_attribute_name(:due_date)      %>: <%= issue.due_date      %>
+<%= WorkPackage.human_attribute_name(:estimated_hours)%>: <%= issue.estimated_hours %>
+
+<% if Setting.work_package_done_ratio != 'disabled' %>
+  <%= WorkPackage.human_attribute_name(:done_ratio)    %>: <%= issue.done_ratio    %>
+<% end %>
```

Comment:
@NobodysNightmare this will create a blank line between estimated_hours and done_ratio which shouldn't be necessary. BTW: This is probably also true for the custom fields.